config/application.yml.default

```diff
@@ -500,7 +500,6 @@ test:
   database_worker_jobs_username: ''
   database_worker_jobs_host: ''
   database_worker_jobs_password: ''
-  doc_auth_enable_presigned_s3_urls: true
```

Comment:
I think this breaks a test assumption that the specs have, the analytics spec is looking for async doc upload:

```
  Expected that FakeAnalytics would have received event "Frontend: IdV: document capture async upload encryption"
```

Reply:
What is the motivation for this change? I assume that subsequent changes to `spec/features/idv/analytics_spec.rb` are attempts to compensate for the breaking assumption that Zach mentions.